app/frontend/src/features/profile/edit/EditHostingPreference.test.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  render,
+  screen,
+  waitFor,
+  queryByAttribute,
+  fireEvent,
+  act,
+} from "@testing-library/react";
+import { Route, Switch } from "react-router-dom";
+import { addDefaultUser, MockedService } from "../../../test/utils";
+import wrapper from "test/hookWrapper";
+import EditHostingPreference from "./EditHostingPreference";
+import { getUser } from "test/serviceMockDefaults";
+import { service } from "service";
+import { profileRoute } from "routes";
+
+const MockProfilePage = "Mock Profile Page";
```

Comment:
```suggestion
const mockProfilePage = "Mock Profile Page";
```
We tend to start variable names in upper case only if it's for actual React components